### PR TITLE
[coati] add vf_coef argument for PPOTrainer

### PR DESCRIPTION
**Pull request by zhang-yi-chi**
## 📌 Checklist before creating the PR

- [x] I have created an issue for this PR for traceability
- [x] The title follows the standard format: `[doc/gemini/tensor/...]: A concise description`
- [ ] I have added relevant tags if possible for us to better distinguish different PRs

## 🚨 Issue number

#3317 

## 📝 What does this PR do?

1. Adding an argument vf_coef for value loss which is common in standard PPO implementation.
2. Setting default multiplier of value loss to 0.5 as shown in the original paper.

## 💥 Checklist before requesting a review

- [x] I have linked my PR to an issue ([instruction](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue))
- [x] My issue clearly describes the problem/feature/proposal, with diagrams/charts/table/code if possible
- [x] I have performed a self-review of my code
- [ ] I have added thorough tests.
- [ ] I have added docstrings for all the functions/methods I implemented

## ⭐️ Do you enjoy contributing to Colossal-AI?

- [x] 🌝 Yes, I do.
- [ ] 🌚 No, I don't.

Tell us more if you don't enjoy contributing to Colossal-AI.

### CODE REVIEW

File: applications/Chat/coati/models/loss.py [hunk 1]
```diff
@@ -65,7 +65,7 @@ def forward(self,
         surr2 = (values - reward)**2
         loss = torch.max(surr1, surr2)
         loss = loss.mean()
-        return loss
```
Comment (ht-zhou) [inline applications/Chat/coati/models/loss.py:68]: Why to return 0.5*loss ?

Comment (zhang-yi-chi) [reply applications/Chat/coati/models/loss.py:68]: I hope to follow  a common implementation before value's MSE loss, as shown in

**Review by ht-zhou — APPROVED** [review]
